notebooks/002-openvino-api/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenVINO API tutorial
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F002-openvino-api%2F002-openvino-api.ipynb)
+
+![coco image](data/classification.jpg)
```

Comment:
@Debskij please use githubusercontent URL for images like this: https://user-images.githubusercontent.com/15709723/127032784-8846df6f-0bfb-44ce-8920-76bcc0b5199e.jpg

![classification](https://user-images.githubusercontent.com/15709723/127032784-8846df6f-0bfb-44ce-8920-76bcc0b5199e.jpg)

Reply:
Done